addons/binding/create_openhab_binding_skeleton.sh

```diff
@@ -1,23 +1,30 @@
 #!/bin/bash
 
+[ $# -lt 2 ] && { echo "Usage: $0 <BindingIdInCamelCase> <Author>"; exit 1; }
+
+bindingVersion=2.3.0-SNAPSHOT
+archetypeVersion=0.10.0-SNAPSHOT
+
 camelcaseId=$1
-[ $# -eq 0 ] && { echo "Usage: $0 <BindingIdInCamelCase>"; exit 1; }
+id=`echo $camelcaseId | tr '[:upper:]' '[:lower:]'`
 
-id=`echo $camelcaseId | tr '[:upper:]' '[:lower:]'` 
+author=$2
 
 mvn -s ../archetype-settings.xml archetype:generate -N \
   -DarchetypeGroupId=org.eclipse.smarthome.archetype \
   -DarchetypeArtifactId=org.eclipse.smarthome.archetype.binding \
-  -DarchetypeVersion=0.10.0-SNAPSHOT \
+  -DarchetypeVersion=$archetypeVersion \
   -DgroupId=org.openhab.binding \
   -DartifactId=org.openhab.binding.$id \
   -Dpackage=org.openhab.binding.$id \
-  -Dversion=2.3.0-SNAPSHOT \
+  -Dversion=$bindingVersion \
   -DbindingId=$id \
   -DbindingIdCamelCase=$camelcaseId \
   -DvendorName=openHAB \
-  -Dnamespace=org.openhab
+  -Dnamespace=org.openhab \
+  -Dauthor="$author"
 
 directory=`echo "org.openhab.binding."$id/`
```

Comment:
It's not part of you changes, but the assignment can be simplified by removing echo:

`directory="org.openhab.binding.$id/"`